docs/source/02_installation.md

```diff
@@ -1,4 +1,4 @@
-# Building ADAM from Source {#build-from-source}
+# Building ADAM from Source
```

Comment:
For now, I would consider the output of the pandoc scripts (`docs/build.sh`) to be the golden copy.

OOC, did one of us set up the readthedocs page?